### ISSUE TITLE
How to add in Karma?

### ISSUE THREAD

**Issue by VitorLuizC** [ratherblue/eslint-html-reporter#12]
Can anyone help me to include this reporter in karma-runner, i tried a lot, but no great results.

I`d add eslint config and include karma-eslint and eslint-html-reporter on dependencies.

``` js
{
    preprocessors: {
        ...
        "app/**/*.js": ["eslint"]
    },
    plugins: [
        ...
        "karma-eslint"
    ],
    eslint: {
        ... // engine, and options(stopOnError ...)
        format: require("eslint-html-reporter"),
        outputFile: "index.html",
        outputDir: "reports/tests"
    }
```
